pkg/wallet/estimation.go

```diff
@@ -110,13 +113,21 @@ func calcTxBaseSize(
 		insSize + outsSize
 }
 
-func calcTxWitnessSize(inScriptTypes, outScriptTypes []int) int {
+func calcTxWitnessSize(
+	inScriptTypes, inAuxiliaryWitnessSize,
+	outScriptTypes []int,
+) int {
 	insSize := 0
+	auxCount := 0
 	for _, scriptType := range inScriptTypes {
-		if scriptType > P2MS {
```

Comment:
is this breaking any of other project may using this package?

Reply:
the estimation function wasn't exposed before this or #322, so other projects would not be affected by this. Anyway, I changed this specific check to increase the precision of estimation for txs with ins/outs of type p2sh(p2wsh) or p2wsh.